Qpf/Macro/Data.lean

```diff
@@ -121,7 +112,7 @@ def mkHeadT (view : InductiveView) : CommandElabM Name := do
   }
   -- The head type is the same as the original type, but with all constructor arguments removed
   let ctors ← view.ctors.mapM fun ctor => do
-    let declName := Name.replacePrefix view.declName declName ctor.declName
+    let declName := ctor.declName.replacePrefix view.declName declName 
```

Comment:
There seems to be a stylistic inconsistency between lines 68 and 115. In line 68, you call replacePrefix on Name, whereas in line 115, you call replacePrefix on ctor.declName.

Is there a reason to write lines 68 and 115 in different syntaxes?

Reply:
Fixed on line 68 now